e2e/explorer/workflow/revisions.spec.ts

```diff
@@ -0,0 +1,239 @@
+import { Page, expect, test } from "@playwright/test";
+import {
+    createNamespace,
+    deleteNamespace,
+} from "../../utils/namespace";
+
+import {
+    createWorkflow,
+} from "../../utils/node";
+import { faker } from "@faker-js/faker";
+
+let namespace = "";
+let workflow = "";
+const defaultDescription = 'A simple \'no-op\' state that returns \'Hello world!\''
+
+test.beforeEach(async ({ page }) => {
+    namespace = await createNamespace();
+    workflow = await createWorkflow(namespace, faker.git.shortSha() + '.yaml');
+    test.setTimeout(120000)
+});
+
+test.afterEach(async () => {
+    await deleteNamespace(namespace);
+    namespace = "";
+});
+
+const navigateToRevisions = async (page: Page) => {
+    await page.goto(`${namespace}/explorer/workflow/revisions/${workflow}`)
+}
+
+const navigateToWorkflowEditor = async (page: Page) => {
+    await page.goto(`${namespace}/explorer/workflow/active/${workflow}`)
+}
+
+const testEditAndSaveWorkflow = async (page: Page, updatedText?: string): Promise<[string, string]> => {
```

Comment:
I didn't check in detail, but it seems there is a high overlap between the methods here and in `active.spec.ts`.

Could you check if it make sense to just declare them once and import them to the other file? 

Maybe we could also make a new file `e2e/explorer/workflow/utils.ts` for methods that are used in multiple spec files?

(as long as they are specific to the workflow page)